templates/default/etcd.initd.erb

```diff
@@ -0,0 +1,103 @@
+#!/bin/bash
+#
+# etcd      This shell script takes care of starting and stopping
+#             etcd
+#
+# chkconfig: 345 80 30
+# description:  etcd service registry
+
+# processname: etcd
+# pidfile: /var/run/etcd/etcd.pid
+
+# Source function library.
+. /lib/lsb/init-functions
+
+# Source etcd configuration
+if [ -f /etc/default/etcd ] ; then
+ . /etc/default/etcd
+fi
+
+RETVAL=0
+prog="etcd"
+
+start() {
+ # Start daemons.
+ echo -n $"Starting $prog: "
+ PID=`pidof etcd`
+ if [ ! -z $PID ]; then
+   return 2
+ fi
+ /usr/local/bin/etcd <%= @arguments.join("\\\n  ") %> > /var/log/etcd 2> /var/log/etcd.err &
+ RETVAL=$?
+ echo
+ [ $RETVAL -eq 0 ] && touch /var/lock/etcd
+ return $RETVAL
+}
+
+stop() {
+ # Stop daemons.
+ echo -n $"Shutting down $prog: "
+ PID=`pidof etcd`
+ if [ ! -z $PID ]; then
+   kill -9 $PID
```

Comment:
`SIGKILL` isn't the safest way to stop etcd. It looks like there are some [shutdown tasks](https://github.com/coreos/etcd/blob/master/etcdserver/server.go#L422-L486) that occur before the process terminates.